src/models/category.js

```
@@ -0,0 +1,39 @@
+const { CATEGORY } = require('~/consts/models')
+const {
+  regex: { HEX_COLOR_CODE_PATTERN, NAME_PATTERN }
+} = require('~/consts/validation')
+
+const categorySchema = new Schema({
+  name: {
+    type: String,
+    required: [true, FIELD_CANNOT_BE_EMPTY('name')],
+    unique: [true, DOCUMENT_ALREADY_EXISTS('name')],
+    regex: NAME_PATTERN,
+    minLength: [1, FIELD_CANNOT_BE_SHORTER('name', 1)],
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)